### DISCUSSION TITLE
Replace queue and replace and play in album and song view is inconsistent

### DISCUSSION THREAD

**Discussion by GK-774** [jcorporation/myMPD#609] [General]
In the Album view the album play button (if configured for replace and not append) replaces the queue but does not immediately play. (Replace and play would be nice.)
In the search view clicking a song with "replace queue" does not only replace but a replace and play (which is ok, but not what it advertises).
(myMPD 8.1.6)

**Comment by jcorporation** [comment]
The autoplay option should handle this. Replace should only replace the queue and with enabled autoplay mpd begins playing this new songs. For myMPD 8.2.0 all this options are reworked completely and gives more options if used with mpd 0.23.x

**Comment by GK-774** [comment]
Many thanks for the hint with "autoplay" - yes this is nice with album play!

**Comment by GK-774** [comment]
Ok, autoplay of course now starts playing even when I just append an ablbum to the queue and mpd was stopped. I can live with that, but not ideal because it´s unexpected and unintended. Personally I think more options is not the answer, it´s consistence. If the album button says "play" it should play. If a field says "replace" it should just do that.
